dashboard/app/controllers/registrations_controller.rb

```diff
@@ -23,18 +23,22 @@ def update
         @user.update_without_password(update_params(params))
       end
 
+    account_update_response(successfully_updated, @user)
```

Comment:
The question of why user was looked up in the first place is a good one, I wasn't sure, so I didn't address it then (though I did copy it), but I don't see any reason why it should need to be looked up. I'll clean it up in both places.

And you're also correct, it doesn't need to be passed in.